weave/trace/weave_client.py

```diff
@@ -745,16 +790,46 @@ def get(self, ref: ObjectRef) -> Any:
     def get_calls(
         self,
         filter: CallsFilter | None = None,
```

Comment:
nit: Would put a `*` after self to force users to enter these as kwargs

Reply:
ah right, perfect.